lib/filter/buddy.js

```diff
@@ -110,7 +111,19 @@ exports.update = function(json) {
       json[buddy].status_ailments.push(ailments[status].id);
       statusAilmentDesc.push(ailments[status].description);
     };
-
+    
+    /**
+     * Hit-all
+     * Overrides the record materia with Zealot and adds the Zealot Attack property
+     * Attack CAN be modified to do several things - hit more than once, etc.
+     */
+  	if (config.get('buddy.attack.multiHit')) {
+	    var materia = recordMateria["ZEALOT"];
+	    var materiaatk = recordMateria["ZEALOTATK"];
+	    json[buddy].materias = materia;
```

Comment:
This is currently by design. We'd need to confirm how we want to handle the config file to check and see if the selected materia has an attack command, and then modify the attack command to use target  mode 6. I wasn't really sure how you wanted to handle that, and since multi-hit pretty much requires the zealot materia, it was hard coded. 

The future plan would be to change it to a config.get(buddy.materias) list, which i started to make, but ran into the problem above (how do you determine if a selected materia has an attack associated?)